### PR TITLE
LPD-25987 - Create a base test case to use for upgrade processes that need to test missing ctCollectionId

### PR DESCRIPTION
**Pull request by david-truong**

### CODE REVIEW

**Review by achaparro — COMMENTED** [review]
Hi @david-truong,

I have reviewed what impacted more to the Upgrade framework. Look good to me but I would do one main change, see my comments and let me know what you think.

Thanks.

File: modules/apps/change-tracking/change-tracking-test-util/src/main/java/com/liferay/change/tracking/test/util/BaseCTUpgradeProcessTestCase.java [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+import org.junit.Test;
+
+/**
+ * @author David Truong
+ */
+public abstract class BaseCTUpgradeProcessTestCase {
+
+	@Before
+	public void setUp() throws Exception {
+		group = GroupTestUtil.addGroup();
```
[the remark this answers is not in the document]
Comment (david-truong) [reply modules/apps/change-tracking/change-tracking-test-util/src/main/java/com/liferay/change/tracking/test/util/BaseCTUpgradeProcessTestCase.java:42]: we probably can remove it

**Comment by david-truong** [comment]
@achaparro i added your suggestions and also I made the fetchCTModel optional.

**Comment by david-truong** [comment]
ci:test:sf

**Comment by david-truong** [comment]
ci:report:2571141354

**Comment by david-truong** [comment]
ci:report:2571141341

**Comment by david-truong** [comment]
ci:forward:force

**Pull request closed without merging**